packages/connect-query/package.json

```diff
@@ -11,8 +11,8 @@
   "scripts": {
     "clean": "rm -rf ./dist/*",
     "build": "npm run build:cjs && npm run build:esm && npm run build:proxy",
-    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
-    "build:esm": "tsc --project tsconfig.build.json --module ES2015 --verbatimModuleSyntax --outDir ./dist/esm --declaration --declarationDir ./dist/esm",
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --moduleResolution node --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
+    "build:esm": "tsc --project tsconfig.build.json --module Node16 --verbatimModuleSyntax --outDir ./dist/esm --declaration --declarationDir ./dist/esm",
```

Comment:
Can we remove `--module Node16` here, since it's already set in tsconfig? I think we can also add verbatimModuleSyntax to tsconfig, and use `--verbatimModuleSyntax false` in `build:cjs`. The idea is that we do a standard compile for ESM, and make adjustments only for CJS.